app/views/galleries/_single.haml

```diff
@@ -2,23 +2,25 @@
 - attrs = {class: 'section-ordered', data: {id: character_gallery.id, order: character_gallery.section_order}} if local_assigns[:character_gallery]
 %tr.gallery-header{**attrs}
   %th.padding-10{class: (klass if defined? klass)}
-    = link_to gallery.name, gallery_path(gallery), class: 'gallery-title'
-    - if gallery.user_id == current_user.try(:id) && !defined? skip_forms
-      = link_to add_gallery_path(gallery), class: 'gallery-add' do
+    - link = gallery ? gallery_path(gallery) : user_gallery_path(id: 0, user_id: @user.id)
+    = link_to gallery ? gallery.name : 'Galleryless icons', link, class: 'gallery-title'
+    - if is_owner && !skip_forms
```

Comment:
Do we always define `skip_forms`? Might be worth using `local_assigns[:skip_forms]`

Reply:
I recall going through and making it so we _did_ define it everywhere this got called from so I could get rid of the `!defined`

Reply:
Ah, I see – cool!